### ISSUE TITLE
P3A traffic isn't sent to the p3a-json.brave.com endpoint

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-browser#23754]
## Description 

Found while testing https://github.com/brave/brave-browser/issues/23147

P3A traffic isn't sent to the `p3a-json.brave.com` endpoint, This is  a regression issue after removing [p3a.brave.com](http://p3a.brave.com/) via https://github.com/brave/brave-browser/issues/23147

## Steps to Reproduce

   1. Clean profile `1.41.82`
   2. Open network Charles network monitoring tool
   3. Launch brave beta `1.41.81`
   4. Waited for almost 1 hour, traffic is not sent to `p3a-json.brave.com` endpoint, I could see only one metric data listed

## Actual result:

P3A traffic isn't sent to the p3a-json.brave.com endpoint

## Expected result:
All the metrics from `brave://local-state` should be shown in the `p3a-json.brave.com` endpoint

## Reproduces how often: 

100%

## Brave version (brave://version info)

Brave | 1.41.82 Chromium: 103.0.5060.66 (Official Build) beta (64-bit)
-- | --
Revision | 20b1569438a85e631d15e83eb355e3e326e5da6f-refs/branch-heads/5060@{#1066}
OS | Windows 10 Version 21H2 (Build 19044.1766)

## Version/Channel Information:

- Can you reproduce this issue with the current release? No
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

## Miscellaneous Information:

cc: @brave/qa-team @rillian @porteron @mattmcalister @rebron 

The issue is reproduced by @rillian and @stephendonner, thanks both!

**Comment by GeetaSarvadnya** [comment]
Please verify the P2A metrics as well once the issue is fixed. Looks like P2A metrics are also affected by this issue. https://github.com/brave/brave-browser/issues/22079#issuecomment-1170016415

**Comment by kjozwiak** [comment]
Closing as https://github.com/brave/brave-core/pull/14015 was merged into `1.41.x`. The above will require `1.41.93` or higher for `1.41.x` verification 👍

**Comment by GeetaSarvadnya** [comment]
Verification PASSED on

```
Brave | 1.41.93 Chromium: 103.0.5060.114 (Official Build) (64-bit)
-- | --
Revision | a1c2360c5b02a6d4d6ab33796ad8a268a6128226-refs/branch-heads/5060@{#1124}
OS | Windows 10 Version 21H2 (Build 19044.1766)
```
 Basically we ensured P3A and P2A works as it works in `1.40.x` due to https://github.com/brave/brave-core/pull/14015

- Confirmed that P3A traffic is sent to the endpoint `p3a-json.brave.com` in 1.41.x
- Confirmed that all the P3A metrics are sent to the json format to `p3a-json.brave.com` in 1.41.x
- Confirmed that after Revert stopping P3A protobuf submission https://github.com/brave/brave-core/pull/14015, the traffic is sent to the old `p3a.brave.com` endpoint in the encoded format in 1.41.x
- Confirmed P2A metrics data sent correctly in both old `p2a.brave.com` and new json `p2a.json.brave.com` formats in 1.41.x
- Confirmed brave://local-state file has `6` records for p2a and same is shown in both p2a-json.brave.com and p2a.brave.com endpoints
- Confirmed brave://local-state file has `47` records for p3a and same is shown in both p3a-json.brave.com and p3a.brave.com endpoints 

p3a data in brave://local-state | p3a.brave.com | p3a-json.brave.com
-- | -- | --
![image](https://user-images.githubusercontent.com/38657976/177518513-38788cb9-02dc-4eb1-86b0-e11513e3112c.png) | ![image](https://user-images.githubusercontent.com/38657976/177516112-ed8b442d-1bb7-459d-aca5-abaec8514e58.png) | ![image](https://user-images.githubusercontent.com/38657976/177516015-87159b0f-9163-4244-9efc-318846cbdbf7.png)

p2a data in brave://local-state | p2a.brave.com | p2a-json.brave.com
-- | -- | --
![image](https://user-images.githubusercontent.com/38657976/177516599-3cb745fe-d183-4af5-938a-713e4df94e18.png) | ![image](https://user-images.githubusercontent.com/38657976/177516354-d4d275eb-c7c9-4079-8354-ece1190c8919.png) | ![image](https://user-images.githubusercontent.com/38657976/177516288-bc8047ad-447d-4906-9856-c5b88314f82d.png)
